[16.0][ADD] purchase_currency (#2508)
OCA/purchase-workflow: purchase_currency/readme/CONFIGURE.rst

Go to the Settings -> Purchase:

> - Select a currency avaibale for the company in the "Default Purchase Currency" field

Reviewer: ```suggestion
- Set "Default Purchase Currency" with an available currency
```
